README.md

```diff
@@ -17,17 +18,59 @@ It supports several languages:
 * Rust
 * Typescript
 
-## How to build it
+## Supported Metrics
 
-```console
-# Fetch the tree-sitter dependencies
-git submodule init
-git submodule update --recursive
+- CC
+- SLOC
+- LLOC
+- HALSTEAD
+- NEXITS
+- NARGS
+
+## How to get metrics
+
+You can get metrics in your shell in this way
+
+```
+./rust-code-analysis --metrics --paths /your/path/to/a/file
+```
+
+or as a `json` file
+
+```
+./rust-code-analysis --metrics --output /path/to/new/json/file --paths /your/path/to/a/file
+```
+
```

Comment:
Also mention that the tool can be run as a HTTP service this way: `rust-code-analysis --serve --port 9090`.

Reply:
Done. How are the metrics displayed? I haven't added that information

Reply:
The service is just offering a REST API, it's basically a (probably faster) alternative to running the tool from the shell.

(Faster because you don't have to restart the process multiple times)

Reply:
Ok, thanks for the info, so I guess I can leave as it is now